src/styles/blocks/card.scss

```
@@ -0,0 +1,114 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 200px;
+  padding: 32px 16px 0;
+
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  border-radius: 5px;
+  border: 1px solid $grey;
+  transition: transform 0.3s;
```

Move it to the vairable & reuse it
```suggestion
  transition: transform ...;
```